build-linux/make_package.sh

```diff
@@ -96,24 +96,30 @@ mkdir -p data/etc/
 cp ../conf/pdagent.conf data/etc/
 
 if [ "$pkg_type" = "deb" ]; then
-    _PY_SITE_PACKAGES=data/usr/lib/python2.7/dist-packages
+    _PY27_SITE_PACKAGES=data/usr/lib/python2.7/dist-packages
+    _PY3_SITE_PACKAGES=data//usr/lib/python3/dist-packages
 else
-    _PY_SITE_PACKAGES=data/usr/lib/python2.6/site-packages
     _PY27_SITE_PACKAGES=data/usr/lib/python2.7/site-packages
+    _PY36_SITE_PACKAGES=data/usr/lib/python3.6/site-packages
+    _PY37_SITE_PACKAGES=data/usr/lib/python3.7/site-packages
 fi
 
 echo = python modules...
-mkdir -p $_PY_SITE_PACKAGES
+mkdir -p $_PY27_SITE_PACKAGES
```

Comment:
Question: If we're looking to use Python3 why are we defaulting to creating a directory for the 2.7 packages?

Reply:
For backward compatibility -- we may find some people stuck on python 2 for a while.

Reply:
Ah gotcha. Thanks!